common/djangoapps/student/models.py

```diff
@@ -288,6 +288,8 @@ class Meta(object):
         blank=True, null=True, max_length=6, db_index=True,
         choices=LEVEL_OF_EDUCATION_CHOICES
     )
+    profession = models.TextField(blank=True, null=True)
```

Comment:
Do you need more than 255 characters? If not, use a `CharField`.

Reply:
This will be solved when I remove specialty and profession as their own columns and save them in the UserProfile.meta column